spec/string-indices-spec.js

```diff
@@ -0,0 +1,17 @@
+const chai = require( 'chai' )
+const { expect } = require( 'chai' )
+const {stringIndices} = require( '../src/string-indices' )
+
+describe( 'stringIndices', () => {
+  'use strict'
+
+  it( 'exists', () => {
+    expect( stringIndices ).to.be.a( 'function' )
+  })
+  it('returns an empty string if the supplied index is longer or shorter than the length of the string.', () => {
```

Comment:
If it's "shorter than the length of the string," wouldn't that be the expected input? Is this meant to be "a negative index"? (e.g. `...supplied index is a negative number or larger than the string length`)